src/js-for-coding-test/PART-03/13-DFS_BFS-문제/19.js

```diff
@@ -0,0 +1,74 @@
+const divide = (a, b) => {
+  const result = Math.floor(Math.abs(a) / b);
+  const sign = Math.abs(a) === a ? 1 : -1;
+
+  return result === 0 ? 0 : result * sign;
+};
```

Comment:
```suggestion
const divide = (a, b) => parseInt(a/b);
```
이렇게 사용하셔도 되지 않을까 싶습니다 😃

Reply:
`parseInt` 보다 `Number`를 쓰는게 좋다는 생각에, `parseInt`는 전혀 안쓰고 있었는데, 여기서는 딱 맞게 사용할 수 있겠네요 :+1: 
감사합니다~

Reply:
`const divide = (a, b) => parseInt(a/b);` 로 하니, `a === -1`, `b === 0` 일 때 `-0` 이 나와서

`const divide = (a, b) => parseInt(a/b) || 0;`  으로 수정했습니다 :smile:

Reply:
-0도 고려가 필요하군요 좋은 예시 감사합니당